util/setup.js

```diff
@@ -8,11 +8,12 @@ let baseConfig = fs.readFileSync('./util/setup_base.js', 'utf8');
 const defaultSettings = `{
```

Comment:
Is there a reason for defualtSettings in both util/setup_base.js and this file?

Reply:
They exist there to allow for the resetting of the default settings. setup_base.js is the file that eventually gets written as config.js.